conda_build/index.py

```diff
@@ -1430,48 +1233,22 @@ def _add_extra_path(extra_paths, path):
         _add_extra_path(extra_paths, join(subdir_path, REPODATA_JSON_FN + '.bz2'))
         _add_extra_path(extra_paths, join(subdir_path, REPODATA_FROM_PKGS_JSON_FN))
         _add_extra_path(extra_paths, join(subdir_path, REPODATA_FROM_PKGS_JSON_FN + '.bz2'))
-        _add_extra_path(extra_paths, join(subdir_path, "repodata2.json"))
+        # _add_extra_path(extra_paths, join(subdir_path, "repodata2.json"))
```

Comment:
why is repodata2.json getting removed?

Reply:
It has never been used. We can bring it back when we do namespaces. I think enough about the repodata will change with sharding that this stuff would need major rework anyway.